rocky/katalogus/templates/plugin_deep_link_detailed_list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "layouts/base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+    {% include "header.html" %}
+
+    <main id="main-content">
+        <section>
+            {{ ooi }}
+            <div>
+                {% include "partials/ooi_head.html" with ooi=ooi view="deep_links" %}
+
+                <h2>{% translate "Deep Links" %}</h2>
+                <p class="emphasized">
+                    {% translate "Showing a list of all deep links" %}
+                </p>
+                <a class="button"
```

Comment:
We should check for a new permisson to see if the user has these rights as it affects the whole organization